src/services/goToDefinition.ts

```diff
@@ -312,7 +315,20 @@ function getDefinitionFromObjectLiteralElement(typeChecker: TypeChecker, node: N
     if (element) {
         const contextualType = element && typeChecker.getContextualType(element.parent);
         if (contextualType) {
-            return flatMap(getPropertySymbolsFromContextualType(element, typeChecker, contextualType, /*unionSymbolOk*/ false), propertySymbol => getDefinitionFromSymbol(typeChecker, propertySymbol, node));
+            let properties = getPropertySymbolsFromContextualType(element, typeChecker, contextualType, /*unionSymbolOk*/ false);
+            if (properties.length === 1) {

```

Comment:
Is there no way that this would ever apply if there were more than one symbol here? Like, can the definition be both the constraint but also something else via a union?